### DISCUSSION TITLE
Upgrading Xamarin from CSLA 5.1.0 to 5.5.2 Could not load file or assembly System.Runtime.Loader

### DISCUSSION THREAD

**Discussion by wilfreds** [MarimerLLC/csla#2716] [Questions]
Hi,

Trying to upgrade a Xamarin project from CSLA 5.1.0 to 5.5.2. The 5.1.0 project is running fine.

Server is running Framework 4.8
App is running Xamarin Forms 4.8.0.1821

When we try to run the app we get:
`System.IO.FileNotFoundException: 'Could not load file or assembly 'System.Runtime.Loader, Version=4.0.0.0, Culture=neutral, PublicKeyToken=b03f5f7f11d50a3a' or one of its dependencies.'`

I noticed that when updating the client dll, that contains the CSLA Bussiness logic, from CSLA 5.1 to 5.5, with nuget, is says:
```
Library.Xam.Standard.Client
Updates:
   Csla.5.1.0 -> Csla.5.5.2
   Microsoft.Extensions.Configuration.B.1.1 -> Microsoft.Extensions.Configuration.3.1.9
   Microsoft.Extensions.Configuration.Abstractions.3.1.1 -> Microsoft.Extensions.Configuration.Abstractions.3.1.9 
   Microsoft.Extensions.Configuration.Binder.3.1.1 -> Microsoft.Extensions.Configuration.Binder.3.1.9 
   Microsoft.Extensions.Dependencylnjection.3.1.1 -> Microsoft.Extensions.Dependencylnjection.3.1.9
   Microsoft.Extensions.Dependencylnjection.Abstractions.3.1.1 -> Microsoft.Extensions.Dependencylnjection.Abstractions.3.1.9 
   Microsoft.Extensions.Primitives.3.1.1 -> Microsoft.Extensions.Primitives.3.1.9 System.ComponentModel.Annotations.4.6.0 -> 
   System.ComponentModel.Annotations.4.7.0 System.Runtime.CompilerServices.Unsafe.4.7.0 -> 
   System.Runtime.CompilerServices.Unsafe.4.7.1
Installing:
   System. Runtime.Loader.4.3.0
```

The target framework for Library.Xam.Standard.Client is .NET Standard 2.1

So, it looks like System.Runtime.Loader is now needed. But it breaks Xamarin apps.

Or is it anything we can do to fix this?

Best regards,
Wilfred

**Comment by rockfordlhotka** [comment]
Probably this issue? https://github.com/MarimerLLC/csla/issues/2715
